### PR TITLE
Add new Getting Started Guide

### PR DESCRIPTION
**Pull request by soerenmartius**
New quickstart guide for the Terramate CLI documentation that incorporates features of Terramate v0.4.4.

### CODE REVIEW

File: docs/cli/getting-started/index.md [hunk 1]
```diff
@@ -1,323 +1,505 @@
...
+## Prerequisites
 
-Next, navigate `"cd"` to the new directory and run:
+You will need the following tooling installed:
 
-```bash
-$ terramate create mysite
-```
+- Any recent version of the `git` command line tool
+- Any current version of [Terraform](https://developer.hashicorp.com/terraform/install)
```
Comment (wmalik) [inline docs/cli/getting-started/index.md:43]:
```suggestion
- A recent version of [Terraform](https://developer.hashicorp.com/terraform/install)
```

File: docs/cli/getting-started/index.md [hunk 2]
```diff
@@ -1,323 +1,505 @@
...
-```
+## Prerequisites
 
-Next, navigate `"cd"` to the new directory and run:
+You will need the following tooling installed:
 
-```bash
-$ terramate create mysite
-```
+- Any recent version of the `git` command line tool
```
Comment (wmalik) [inline docs/cli/getting-started/index.md:42]:
```suggestion
- A recent version of the `git` command line tool
```

File: docs/cli/getting-started/index.md [hunk 3]
```diff
@@ -1,323 +1,505 @@
...
-```bash
-$ terramate list --changed
-dev/mysite
 ```
+:::
+
+## Conclusion
+
+We hope this tutorial has helped you grasp the basics of Terramate.
+Here's a summary of what we learned in just 5 minutes:
```
Comment (wmalik) [inline docs/cli/getting-started/index.md:477]:
```suggestion
Here's a summary of what we learned:
```

File: docs/cli/getting-started/index.md [hunk 4]
```diff
@@ -1,323 +1,505 @@
...
-    resource "local_file" "mysite" {
-      filename = "/tmp/tfmysite/${terramate.stack.path.relative}/index.html"
-...
+::: code-group
+```sh [shell]
+terramate run terraform init
 ```
 
-To execute Terraform, navigate `"cd"` to `prod/mysite` and `dev/mysite` and run the necessary Terraform commands.
+```sh [output]
```
Comment (wmalik) [inline docs/cli/getting-started/index.md:281]: The output looks a bit strange (the first word of each line is highlighted, see screenshot below). We could drop the `sh` from the code block to not use any colors in the output? 🤔 

image

**Review by wmalik — APPROVED** [review]
Nice one, I really like the guide. I have some nitpicky comments, feel free to address them in a different PR if you want this merged soon.

File: docs/cli/getting-started/index.md (hunk 1 above)
Comment (soerenmartius) [reply docs/cli/getting-started/index.md]: resolved

File: docs/cli/getting-started/index.md (hunk 4 above)
Comment (soerenmartius) [reply docs/cli/getting-started/index.md]: changed this to text output instead - very good catch!

File: docs/cli/getting-started/index.md (hunk 2 above)
Comment (soerenmartius) [reply docs/cli/getting-started/index.md]: resolved

**Review by wmalik — APPROVED** [review]

**Pull request merged**